Cargo.toml

```diff
@@ -1,13 +1,13 @@
 [package]
 name = "sprocket"
 version = "0.11.0"
-description = "A command line tool for working with Workflow Description Language (WDL) files"
-authors = ["Clay McLeod <clay.l.mcleod@gmail.com>"]
+description = "A command line tool for working with Workflow Description Language (WDL) documents"
+authors = ["The Rust WDL project developers"]
```

Comment:
This matches the Cargo.toml in the wdl repo, but is it what we want? I think a better author line would either `St Jude Rust Labs` or `Sprocket project developers`. "Rust WDL project" is just kind of generic 🤷‍♀️